addons/udes_stock/data/routes.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<odoo>
+  <data>
+
+    <!-- Create "Goods In, Putaway" route -->
+    <record id="route_in" model="stock.location.route">
+      <field name="name">Inbound</field>
+      <field name="sequence" eval="10"/>
+      <field name="warehouse_selectable" eval="True"/>
+      <field name="warehouse_ids" eval="[(6, False, [ref('stock.warehouse0')])]"/>
+    </record>
+
+    <!-- Create "Pick, Goods Out" route -->
+    <record id="route_out" model="stock.location.route">
+      <field name="name">Outbound</field>
+      <field name="sequence" eval="10"/>
+      <field name="warehouse_selectable" eval="True"/>
+      <field name="warehouse_ids" eval="[(6, False, [ref('stock.warehouse0')])]"/>
+    </record>
+
+    <record id="procurement_in_putaway" model="stock.rule">
+      <field name="name">Putaway</field>
+      <field name="route_id" eval="route_in"/>
+      <field name="sequence" eval="20"/>
+      <field name="location_src_id" ref="location_input_received"/>
+      <field name="location_id" ref="stock.stock_location_stock"/>
+      <field name="procure_method">make_to_order</field>
+      <field name="action">push</field>
+      <field name="picking_type_id" ref="picking_type_putaway"/>
+    </record>
+
+    <!-- Create "Pick" rule -->
+    <record id="procurement_out_pick" model="stock.rule">
+      <field name="name">Pick</field>
+      <field name="route_id" eval="route_out"/>
+      <field name="sequence" eval="10"/>
+      <field name="location_src_id" ref="stock.stock_location_stock"/>
+      <field name="location_id" ref="location_goods_out_lane"/>
+      <field name="procure_method">make_to_stock</field>
+      <field name="action">pull</field>
+      <field name="picking_type_id" ref="picking_type_pick"/>
+    </record>
+
+    <!-- Create "Goods Out" rules -->
+    <record id="procurement_out_out" model="stock.rule">
+      <field name="name">Goods Out</field>
+      <field name="route_id" eval="route_out"/>
+      <field name="sequence" eval="30"/>
+      <field name="location_src_id" ref="location_goods_out_lane"/>
+      <field name="location_id" ref="location_trailer"/>
+      <field name="procure_method">make_to_order</field>
+      <field name="action">pull</field>
+      <field name="picking_type_id" ref="stock.picking_type_out"/>
+    </record>
+
+    <!-- Create "Trailer Dispatch" procurement rules -->
```

Comment:
Super nit: Remove `procurement`. The model was previously called `procurement.rule` but that is soooo Odoo 11. 😅